### PR TITLE
ECHO_SQL env variable

### PR DESCRIPTION
**Pull request by Idzikowski-Casey**
Allows users to externally set whether or not to see the SQLAlchemy queries being run on database, log level `INFO`.

### CODE REVIEW

File: backend/sparrow/logs/config.py [hunk 1]
```diff
@@ -35,7 +37,7 @@
         "sqlalchemy.engine": {
             "handlers": ["default"],
             "level": "INFO",
-            "propagate": False,
+            "propagate": ECHO_SQL,
```
Comment (davenquinn) [inline backend/sparrow/logs/config.py:40]: Seems like we should pick _either_ the engine config or the logging config? Unless I am missing something...

File: backend/sparrow/settings.py [hunk 2]
```diff
@@ -2,9 +2,12 @@
 
 config = Config()
 
+echo = config("SPARROW_ECHO_SQL", default="false").lower() in ["true"] 
+
 LAB_NAME = config("SPARROW_LAB_NAME", default="Test lab")
 DATABASE = config("SPARROW_DATABASE", default="postgresql+psycopg2:///sparrow")
 BASE_URL = config("SPARROW_BASE_URL", default="/")
+ECHO_SQL = echo
```
Comment (davenquinn) [inline backend/sparrow/settings.py:10]: Put this inline and use starlette's built in casting functionality. Don't hand-roll it! You won't catch other acceptable ways to define true (e.g., `1`, `True`, etc.).

```ECHO_SQL = config('ECHO_SQL', cast=bool, default=False)```

**Review by davenquinn — CHANGES_REQUESTED** [review]
Small changes needed, looks good overall

File: backend/sparrow/logs/config.py (hunk 1 above)
Comment (Idzikowski-Casey) [reply backend/sparrow/logs/config.py]: You're right, and I can't seem to get the SQL to log using this only, so the env variable will be read in at engine configuration.

**Pull request merged**